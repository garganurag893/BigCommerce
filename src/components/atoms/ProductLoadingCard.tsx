const ProductLoadingCard = () => (
  <li className="mx-3 my-6 max-w-xxs sm:w-72 rounded overflow-hidden hover:shadow-lg hover:scale-105 duration-300">
    <div className="animate-pulse">
      <div className="w-full bg-slate-400 h-64 rounded"></div>
      <div className="py-2">
        <div className="w-full h-6 bg-slate-400 rounded mb-2"></div>
        <div className="w-full h-4 bg-slate-400 rounded"></div>
      </div>
    </div>
  </li>
);

export default ProductLoadingCard;
