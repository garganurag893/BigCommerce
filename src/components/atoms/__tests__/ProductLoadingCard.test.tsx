import { render } from '@testing-library/react'

import ProductLoadingCard from '../ProductLoadingCard'

describe('Test ProductLoadingCard Component', () => {
  it('ProductLoadingCard Snapshot Test', () => {
    const { container } = render(<ProductLoadingCard />)
    expect(container).toMatchSnapshot()
  })
})
