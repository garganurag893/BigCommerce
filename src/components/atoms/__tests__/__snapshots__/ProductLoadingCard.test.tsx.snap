// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ProductLoadingCard Component ProductLoadingCard Snapshot Test 1`] = `
<div>
  <li
    class="mx-3 my-6 max-w-xxs sm:w-72 rounded overflow-hidden hover:shadow-lg hover:scale-105 duration-300"
  >
    <div
      class="animate-pulse"
    >
      <div
        class="w-full bg-slate-400 h-64 rounded"
      />
      <div
        class="py-2"
      >
        <div
          class="w-full h-6 bg-slate-400 rounded mb-2"
        />
        <div
          class="w-full h-4 bg-slate-400 rounded"
        />
      </div>
    </div>
  </li>
</div>
`;
